trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid_parameters.F

```diff
@@ -0,0 +1,108 @@
+! This module defines and instantiates objects
+! for a hybrid type reservoir's
+! parameters/properties. Parameters holds
+! static/unchanging variables that are
+! set when the given reservoir object is
+! initialized/instantiated.
+
+module module_persistence_levelpool_hybrid_parameters
+    use module_reservoir_utilities, only: read_netcdf_lake_id, &
+                                          read_persistence_netcdf_gage_id, &
+                                          read_persistence_netcdf_real_2D_parameters, &
+                                          handle_err
+    use module_reservoir_base
+    use netcdf
+    implicit none
+
+    ! Extend/derive hybrid parameters from the abstract base
+    ! type for reservoir properties.
+    type, extends(reservoir_base_properties_type) :: hybrid_parameters_interface
```

Comment:
Why change the naming from "properties" to "parameters"? 
Again, these make my head hurt. Less is more. I would suggest something like
`type, extends(reservoir_properties_) :: reservoir_properties_hybrid`

Reply:
@jdmattern-noaa will change from 'parameters' to 'properties'. After discussing with Don, we think 'properties' would be more appropriate because it encompasses system properties that belong to the class along with model parameters. For the next version, we could further separate system properties and model parameters. I based the design upon Don's previous work for overland and groundwater, which he says just held model parameters.